pkg/resolver/searchComplete.go

```diff
@@ -122,30 +136,32 @@ func (s *SearchCompleteResult) searchCompleteResults(ctx context.Context) ([]*st
 			srchCompleteOut = append(srchCompleteOut, &prop)
 		}
 	}
-	isNumber := isNumber(srchCompleteOut)
-	if isNumber { //check if valid number
-		isNumber := "isNumber"
-		srchCompleteOutNum := []*string{&isNumber} //isNumber should be the first argument if the property is a number
-		// Sort the values in srchCompleteOut
-		sort.Slice(srchCompleteOut, func(i, j int) bool {
-			numA, _ := strconv.Atoi(*srchCompleteOut[i])
-			numB, _ := strconv.Atoi(*srchCompleteOut[j])
-			return numA < numB
-		})
-		if len(srchCompleteOut) > 1 {
-			// Pass only the min and max values of the numbers to show the range in the UI
-			srchCompleteOut = append(srchCompleteOutNum, srchCompleteOut[0], srchCompleteOut[len(srchCompleteOut)-1])
-		} else {
-			srchCompleteOut = append(srchCompleteOutNum, srchCompleteOut...)
-		}
+	if len(srchCompleteOut) > 0 {
+		//Check if results are date or number
+		isNumber := isNumber(srchCompleteOut)
+		if isNumber { //check if valid number
+			isNumber := "isNumber"
+			srchCompleteOutNum := []*string{&isNumber} //isNumber should be the first argument if the property is a number
+			// Sort the values in srchCompleteOut
+			sort.Slice(srchCompleteOut, func(i, j int) bool {
+				numA, _ := strconv.Atoi(*srchCompleteOut[i])
+				numB, _ := strconv.Atoi(*srchCompleteOut[j])
+				return numA < numB
+			})
+			if len(srchCompleteOut) > 1 {
+				// Pass only the min and max values of the numbers to show the range in the UI
+				srchCompleteOut = append(srchCompleteOutNum, srchCompleteOut[0], srchCompleteOut[len(srchCompleteOut)-1])
+			} else {
+				srchCompleteOut = append(srchCompleteOutNum, srchCompleteOut...)
+			}
 
+		}
+		if !isNumber && isDate(srchCompleteOut) { //check if valid date
+			isDate := "isDate"
+			srchCompleteOutNum := []*string{&isDate}
+			srchCompleteOut = srchCompleteOutNum
```

Comment:
And please variables to match my previous comment.
```suggestion
		if !isNumber && isDate(srchCompleteOut) { //check if valid date
			isDateStr := "isDate"
			srchCompleteOutDate := []*string{&isDateStr}
			srchCompleteOut = srchCompleteOutDate
```